src/DateTime/tests/test_datetime.py

```diff
@@ -683,6 +683,21 @@ def test_security(self):
         self.assertEqual(dt.__roles__, None)
         self.assertEqual(dt.__allow_access_to_unprotected_subobjects__, 1)
 
+    @unittest.skipUnless(PY3K, 'format method is Python 3 only')
+    def test_format(self):
+        dt = DateTime()
+        fmt = '%-d.%-m.%Y %H:%M'
+        result = dt.strftime(fmt)
+        self.assertEqual(result, '{:%-d.%-m.%Y %H:%M}'.format(dt))
+        self.assertEqual(str(dt), '{:}'.format(dt))
+        self.assertEqual(str(dt), '{}'.format(dt))
```

Comment:
This is not well readable, I'd expect something like: (here and for all the other assertions in this test)

```python
self.assertEqual('2022-06-30 08:12:34', '{}'.format(dt))
```

This would require to use a static point in time in line 688 instead of using the current date, but this is fine as we do not test the `now` and `__str__` methods here.

Reply:
Okay, I see, you implemented it while I was writing. 😀